docs/source/entities-advanced.mdx

```diff
@@ -477,29 +477,18 @@ Follow these steps to use the `@override` directive:
       ```graphql {3} title="Billing subgraph"
       type Bill @key(fields: "id") {
         id: ID!
-        amount: Int! @override(from: "Payments") #highlight-line
+        amount: Int! @override(from: "Payments", label: "percent(1)") #highlight-line
       }
       ```
 
     </CodeColumns>
 
-      Applying the `@override` directive tells the router to resolve the `amount` field in the Billing subgraph instead of the Payments subgraph.
+      Applying the `@override` directive tells the router to resolve the `amount` field in the Billing subgraph instead of the Payments subgraph. 
 
+      Adding a `label` argument to the `@override` directive sets the percentage of traffic to direct to the Billing subgraph. Start with a small percentage. Setting `label: "percent(1)"` means that 1 percent of the requests for `amount` are resolved by the Billing subgraph, while the remaining 99 percent are resolved by the Payments subgraph. 
 
 3. Update your router's supergraph schema to migrate to the updated Billing subgraph.
 
-   * Use progressive `@override` by adding a `label` argument to the `@override` directive with the percentage of requests that you want to be resolved by the subgraph. For example:
-
-      ```graphql title="Billing subgraph"
-      type Bill @key(fields: "id") {
-        id: ID!
-        amount: Int! @override(from: "Payments", label: "percent(15)") #highlight-line
-      }
-      ``` 
-
-      Setting `label: "percent(15)"` means that 15 percent of the requests for `amount` are resolved by the Billing subgraph, while the remaining 85 percent are resolved by the Payments subgraph. 
-
-
 When the router receives its updated supergraph schema, it immediately starts resolving the `Bill.amount` field from the Billing subgraph while continuing to resolve `Bill.payment` from the Payments subgraph.
```

Comment:
```suggestion
When the router receives its updated supergraph schema, it starts resolving the `Bill.amount` field from the Billing subgraph approximately 1% of the time, while continuing to resolve it from the Payments subgraph the other 99%.
```
I changed a couple things here. Worth mentioning I'm dropping the callout to `Bill.payment`, I think it's ok not to mention it here. We can keep it if you think it's helpful though.